### ISSUE TITLE
WARNING:root:NaN or Inf found in input tensor.

### ISSUE THREAD

**Issue by merisgong208** [zylo117/Yet-Another-EfficientDet-Pytorch#304]
Very good work of your repo, Zylo117.

I use my own dataset to train, and got the following waring:

.....
Val. Epoch: 0/12. Classification loss: nan. Regression loss: nan. Total loss: nan
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
Step: 9. Epoch: 1/12. Iteration: 3/7. Cls loss: 1.25749. Reg loss: 0.54555. Total loss: 1.80303:  43%|████▎     | 3/7 [01:29<02:05, 31.27s/it]checkpoint...
Step: 13. Epoch: 1/12. Iteration: 7/7. Cls loss: 1.44147. Reg loss: 0.48580. Total loss: 1.92727: 100%|██████████| 7/7 [03:12<00:00, 27.47s/it]
Val. Epoch: 1/12. Classification loss: nan. Regression loss: nan. Total loss: nan
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
Step: 14. Epoch: 2/12. Iteration: 1/7. Cls loss: 1.33619. Reg loss: 0.53487. Total loss: 1.87106:  14%|█▍        | 1/7 [00:35<03:31, 35.22s/it]checkpoint...
Step: 19. Epoch: 2/12. Iteration: 6/7. Cls loss: 1.16544. Reg loss: 0.51639. Total loss: 1.68183:  86%|████████▌ | 6/7 [02:43<00:27, 27.38s/it]checkpoint...
Step: 20. Epoch: 2/12. Iteration: 7/7. Cls loss: 1.65887. Reg loss: 0.43756. Total loss: 2.09643: 100%|██████████| 7/7 [03:09<00:00, 27.11s/it]
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
WARNING:root:NaN or Inf found in input tensor.
...

Platform:
win7
Python3.7
Pytorch1.5

.

**Comment by zylo117** [comment]
overfit.
val loss becomes inf

**Comment by merisgong208** [comment]
@zylo117 I dont think it is overfit, because the warning appears at the very beginning of the train.

It is devergent?

**Comment by zylo117** [comment]
overfitting at the very begining will lead to exploding val loss.

**Comment by merisgong208** [comment]
I think the problem is from my dataset. First, I don't have enough images; Second,  there might be impropers points while converting voc to coco jasons.  Thank you again, zylo.
